hack/e2e-kind-cluster-setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -xe
+
+CNI_VERSION=${CNI_VERSION:-0.4.0}
+OCI_BIN=${OCI_BIN:-docker}
+IMG_REGISTRY=${IMAGE_REGISTRY:-localhost:5000/maiqueb}
```

Comment:
note for a possible follow PR: maybe worth to use generic registry name instead maiqueb ?
beside the latest nits, conceptual lgtm, thanks

Reply:
that'll happen if / when this repo is handed off to the k8snetworkplumbingwg.